### PR TITLE
update several statements

### PR DESCRIPTION
**Pull request by bobotu**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### CODE REVIEW

File: sql-statements/sql-statement-do.md [hunk 1]
```diff
@@ -7,7 +7,13 @@ aliases: ['/docs-cn/dev/reference/sql/statements/do/']
 
 # DO
 
-`DO` 语句用于执行表达式，而不返回结果。在 MySQL 中，`DO` 的一个常见用例是执行存储的程序，而无需处理结果。但是 TiDB 不提供存储例程，因此该功能的使用较为受限。
+`DO` 语句用于执行表达式，但不返回任何结果。大部分情况下，`DO` 相当于不返回结果的 `SELECT expr, ...,`。
+
+> **注意：**
+>
+> `DO` 只能执行表达式，所以不是所有能够用 `SELECT` 的地方都能用 `DO` 替换。例如 `DO id FROM t1` 就是不是合法的 SQL，因为它引用了一张表。
```
Comment (lilin90) [inline sql-statements/sql-statement-do.md:14]:
```suggestion
> `DO` 只能执行表达式，所以不是所有能够用 `SELECT` 的地方都能用 `DO` 替换。例如 `DO id FROM t1` 就是不是合法的 SQL 语句，因为它引用了一张表。
```

File: sql-statements/sql-statement-do.md [hunk 2]
```diff
@@ -7,7 +7,13 @@ aliases: ['/docs-cn/dev/reference/sql/statements/do/']
...
+
+`DO` 在 MySQL 中的一个主要应用场景是存储过程或者触发器。因为 TiDB 当前不支持存储过程和触发器，所以 `DO` 的实际使用场景会比较少。
```
Comment (lilin90) [inline sql-statements/sql-statement-do.md:16]:
```suggestion
`DO` 在 MySQL 中的一个主要应用场景是存储过程或者触发器。因为 TiDB 当前不支持存储过程和触发器，所以 `DO` 的实际使用场景较少。
```

File: sql-statements/sql-statement-set-variable.md [hunk 3]
```diff
@@ -7,16 +7,26 @@ aliases: ['/docs-cn/dev/reference/sql/statements/set-variable/']
...
 
 ![SetStmt](/media/sqlgram/SetStmt.png)
 
+**VariableAssignment:**
+
+![VariableAssignment](/media/sqlgram/VariableAssignment.png)
+
 ## 示例
 
+获取 `sql_mode` 的值
```
Comment (lilin90) [inline sql-statements/sql-statement-set-variable.md:28]:
```suggestion
获取 `sql_mode` 的值：
```

File: sql-statements/sql-statement-set-variable.md [hunk 4]
```diff
@@ -47,6 +57,8 @@ SHOW SESSION VARIABLES LIKE 'sql_mode';
 1 row in set (0.00 sec)
 ```
 
+更新全局的 `sql_mode`
```
Comment (lilin90) [inline sql-statements/sql-statement-set-variable.md:60]:
```suggestion
更新全局的 `sql_mode`：
```

File: sql-statements/sql-statement-set-variable.md [hunk 5]
```diff
@@ -57,6 +69,8 @@ SET GLOBAL sql_mode = 'STRICT_TRANS_TABLES,NO_AUTO_CREATE_USER';
 Query OK, 0 rows affected (0.03 sec)
 ```
 
+检查更新之后的 `sql_mode` 的取值，可以看到 SESSION 级别的值没有更新
```
Comment (lilin90) [inline sql-statements/sql-statement-set-variable.md:72]:
```suggestion
检查更新之后的 `sql_mode` 的取值，可以看到 SESSION 级别的值没有更新：
```

File: sql-statements/sql-statement-set-variable.md [hunk 6]
```diff
@@ -87,6 +101,8 @@ SHOW SESSION VARIABLES LIKE 'sql_mode';
 1 row in set (0.00 sec)
 ```
 
+`SET SESSION` 则可以立即生效
```
Comment (lilin90) [inline sql-statements/sql-statement-set-variable.md:104]:
```suggestion
`SET SESSION` 则可以立即生效：
```

File: sql-statements/sql-statement-show-variables.md [hunk 7]
```diff
@@ -25,90 +25,128 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-variables/']
 
 ## 示例
 
+查看 TiDB 定义的专用系统变量，关于这些变量的含义可以参考 [TiDB 专用系统变量和语法](/tidb-specific-system-variables.md)。
```
Comment (lilin90) [inline sql-statements/sql-statement-show-variables.md:28]:
```suggestion
查看 TiDB 定义的专用系统变量，关于这些变量的含义参见 [TiDB 专用系统变量和语法](/tidb-specific-system-variables.md)。
```

File: sql-statements/sql-statement-show-variables.md [hunk 8]
```diff
@@ -25,90 +25,128 @@ aliases: ['/docs-cn/dev/reference/sql/statements/show-variables/']
...
+
 {{< copyable "sql" >}}
 
 ```sql
 SHOW GLOBAL VARIABLES LIKE 'tidb%';
 ```
 
 ```
```
Comment (lilin90) [inline sql-statements/sql-statement-show-variables.md:36]:
```suggestion
```sql
```

**Comment by bobotu** [comment]
PTAL @lilin90

**Pull request merged**